### ISSUE TITLE
Need help: Looks like `ppc64le_defconfig` missing certain configs related default(virbr0) libvirt bridge network and fails to start the same

### ISSUE THREAD

**Issue by sathnaga** [linuxppc/issues#231]
KVM libvirt default bridge network does not start with below error when kernel built with `ppc64le_defconfig` whereas same kernel built with fedora rawhide(30) kernel it just works fine.
```
error: Failed to start network default
error: COMMAND_FAILED: '/usr/sbin/iptables -w10 -w --table filter --insert LIBVIRT_INP --in-interface virbr0 --protocol tcp --destination-port 67 --jump ACCEPT' failed: iptables: No chain/target/match by that name.
```

Looks like some configs to be enabled, and there are more related configs, needs help around the same.
Below is my upstream libvirt bug report for the same issue, which has details on what are all the missing configs from fedora rawhide.

Background: https://bugzilla.redhat.com/show_bug.cgi?id=1683174#c9

**Comment by mopsfelder** [comment]
`ppc64le_defconfig` is very handful to boot a qemu powernv guest.  We shouldn't bloat it, IMO.

**Comment by sathnaga** [comment]
> `ppc64le_defconfig` is very handful to boot a qemu powernv guest. We shouldn't bloat it, IMO.

True, but the same config file was working in past kernel version 4.20ish etc...
If it going to be one or two I guess it should be fine?
or we can even have another `ppc64le_kvm_defconfig` which can include kvm related configs etc, like `ppc64le_guest_defconfig`...
